ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html

```diff
@@ -1,5 +1,21 @@
 <h2>{{ _("Corrections and notices") }}</h2>
```

Comment:
I wonder if we should wrap these in https://service-manual.ons.gov.uk/design-system/components/details

as well as only display the h2 heading if there either corrections or notices. A small thing, but looks good on the FE ;)